products/eks.md

```diff
@@ -12,32 +12,39 @@ releases:
 -   releaseCycle: "1.22"
     eol: 2023-05-01
     latest: "1.22.9"
-    releaseDate: 2022-04-04
+    releaseDate: 2022-04-05
```

Comment:
The dates are based on the date of the first commit on the EKS documentation that mentions a given release. As such, dates might lag behind by a few days, which seems okay.